src/portalbackend/PortalBackend.DBAccess/Repositories/ServiceAccountRepository.cs

```diff
@@ -144,10 +144,19 @@ public ServiceAccountRepository(PortalDbContext portalDbContext)
                             serviceAccount.OfferSubscription.OfferId,
                             serviceAccount.OfferSubscription.Offer!.OfferTypeId,
                             serviceAccount.OfferSubscription.Offer.Name,
-                            serviceAccount.OfferSubscription.Id)))
+                            serviceAccount.OfferSubscription.Id),
+                    serviceAccount.Identity.LastEditor!.IdentityTypeId == IdentityTypeId.COMPANY_USER
+                        ? serviceAccount.Identity.LastEditor == null
+                            ? null
+                            : new CompanyLastEditorData(
+                                serviceAccount.Identity.LastEditor.CompanyUser!.Lastname!,
+                                serviceAccount.Identity.LastEditor.Company!.Name)
+                        : new CompanyLastEditorData(
+                            serviceAccount.Identity.LastEditor!.CompanyServiceAccount!.Name,
+                            serviceAccount.Identity.LastEditor.Company!.Name)))
```

Comment:
I'd suggest to change the implementation to the following, since it's better readable in my opinion

```suggestion
                            serviceAccount.OfferSubscription.Id),
                    serviceAccount.Identity.LastEditorId == null ? null :
                    new CompanyLastEditorData(
                        serviceAccount.Identity.LastEditor!.IdentityTypeId == IdentityTypeId.COMPANY_USER ? 
                            serviceAccount.Identity.LastEditor.CompanyUser!.Lastname : 
                            serviceAccount.Identity.LastEditor.CompanyServiceAccount!.Name,
                        serviceAccount.Identity.LastEditor.Company!.Name)))
```